app/controllers/mixins/actions/provider_actions/mass_transform.rb

```diff
@@ -0,0 +1,56 @@
+module Mixins
+  module Actions
+    module ProviderActions
+      module MassTransform
+        def vm_transform_mass
+          dialog = Dialog.find_by!(:label => 'Transform VM')
+          @right_cell_text = _("Transform VMs to RHV")
+          if params.key?(:id)
+            provider = ExtManagementSystem.find_by!(:id => params[:id].to_i)
+            dialog_initialize(
+              dialog.resource_actions.first,
+              :header     => @right_cell_text,
+              :target_id  => provider.id,
+              :target_kls => Provider.name
```

Comment:
should this have been `provider.class.name`? If not, maybe use `:target_kls => "Provider"` instead.

Reply:
@himdel I see that `Vm.name` is used here https://github.com/ManageIQ/manageiq-ui-classic/blob/dd0eadbacc34eab74dd1524e7492d8825bdaedbb/app/controllers/mixins/actions/vm_actions/transform.rb#L14 for the same parameter.

Reply:
OK, if this was intentional, no problem there .. just making sure :)

(On the one hand, `Provider.name` sounds like a relatively magical way of saying `"Provider` .. but then on the other, we wouldn't get a nice exception if `Provider` ever gets renamed, so.. (shrug) :).)